src/components/pages/Categories/Management.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <ul class="category">
+    <!-- カテゴリー管理 -->
+    <li class="category-post">
+      <CategoryPost
```

Comment:
[nits]

以下のようになるように修正をお願いします。

```suggestion
      <app-category-post
```

src/components/pages/Articles/List.vue のコンポーネントなどを見ると、`<app-article-list>` のような記述でコンポーネントが使用されていると思います。
アプリケーション全体のコンポーネントを通して基本的に記述の方針が揃っていないと、他の開発メンバーがどっちに合わせればいいのだろう？というような混乱を招く可能性があるためです。